app/models/redis_store.rb

```diff
@@ -2,12 +2,24 @@
 class RedisStore
   extend ActiveModel::Naming
   extend ActiveModel::Callbacks
-  define_model_callbacks :initialize, only: :after
 
   include ActiveModel::Serialization
   include ActiveModel::Validations
   include Virtus.model
-  REDIS_STORE = Redis.current
+
+  define_model_callbacks :initialize, only: :after
+
+  class << self
+    attr_accessor :redis, :ttl
+  end
+
+  def self.redis_store(namespace)
+    @redis ||= Redis::Namespace.new(namespace, redis: Redis.current)
+  end
+
+  def self.default_ttl(ttl)
+    @ttl ||= ttl
+  end
```

Comment:
So the naming conventions might not be clear -- should we maybe use redis_nam_ space so that the class level accessor always requires a namespace?

Why not make the instance variables consistent with the class accessor method name?